src/analyzer/protocol/ssl/Plugin.cc

```diff
@@ -2,8 +2,6 @@
 
 #include "zeek/plugin/Plugin.h"
 
-#include "zeek/zeek-config.h"
-
 #ifndef ENABLE_SPICY_SSL
```

Comment:
Isn't that in zeek-config.h ?

Reply:
That's fair. So seeing `zeek/plugin/Plugin.h` one can assume the `zeek/zeek-config.h` include? It's a bit surprising, but don't have a strong reaction.

Reply:
Yeah, it looks like this:

```
. /Users/tim/Desktop/projects/zeek/src/include/zeek/plugin/Plugin.h
.. /Users/tim/Desktop/projects/zeek/src/include/zeek/ZeekArgs.h
... /Users/tim/Desktop/projects/zeek/src/include/zeek/ZeekList.h
.... /Users/tim/Desktop/projects/zeek/src/include/zeek/List.h
..... /Users/tim/Desktop/projects/zeek/src/include/zeek/util.h
...... /Users/tim/Desktop/projects/zeek/build/zeek/zeek-config.h
```

Reply:
In case you're wondering, `List.h` brings in `util.h` because of `util::safe_malloc`.

Reply:
I am not sure I like depending on that magic include chain to remain stable, especially for a preprocessor macro. My hunch would have also been to keep the `#include "zeek/zeek-config.h"` for `ENABLE_SPICY_SSL`.

Reply:
Oh, I thought `plugin/Plugin.h` directly includes `zeek-config.h`, but hadn't checked. If it's like those 5 layers deep, I'm actually with @bbannier that we should have it explicitly. Or maybe also here do `-DENABLE_SPICY_SSL` on the source file explicitly.

That `-D` approach might be a discussion elsewhere - I don't have an intuition if that's a good pattern or worse, but it feels like a more localized approach.